data/_models/shared/Catalogues.csv

```diff
@@ -1,5 +1,8 @@
 tableName,tableExtends,columnName,columnType,key,required,refSchema,refTable,refLink,refBack,validation,semantics,description,profiles,visible
 Catalogues,,,,,,,,,,,dcat:Catalog,A collection of resources within a network or consortium or about a common topic,DataCatalogueFlat,
+Catalogues,,name,,1,true,,,,,,"dterms:title,dct:title",The name given to the Catalogue,DataCatalogueFlat,
 Catalogues,,network,ref,1,true,,Collections,,,,,Network or consortium that leads this catalogue,DataCatalogueFlat,
 Catalogues,,type,ontology,,true,CatalogueOntologies,CatalogueTypes,,,,dcat:theme,Type of catalogue,DataCatalogueFlat,
-Catalogues,,publisher,ontology,,,CatalogueOntologies,CatalogueTypes,,,,dcat:publisher,Publisher todo should default to molgenis team,DataCatalogueFlat,
\ No newline at end of file
+Catalogues,,description,text,,true,,,,,,"dcat:description,dct:description",Description of this catalogue,DataCatalogueFlat,
+Catalogues,,publisher,ontology,,,CatalogueOntologies,CatalogueTypes,,,,"dcat:publisher,dct:publisher",Publisher todo should default to molgenis team,DataCatalogueFlat,
+Catalogues,,applicable legislation	,ref_array,,true,,Documentation,,,,dcat-ap:applicableLegislation,The legislation that mandates the creation or management of the Catalog,DataCatalogueFlat,
```

Comment:
trailing spaces in columnName? And dcat-ap:applicableLegislation can be changed to dcatap:applicableLegislation to conform to prefix used in dcat-ap

Reply:
Should we add an attribute dcat:dataset to dcat:Catalog, to refer to the Datasets that are part of the Catalogue?
This will conflict however with the setup that we have here with the Catalogues table referencing a Network. We discussed earlier to move the Networks table outside of the Collections again. So for our Networks table we could add a columnName 'datasets', columnType 'refback', refTable 'Collections' or 'Datasets' and have a ref_array from the Datasets/Collections to Networks. Then maybe still keep the Catalogues table to refer to the catalogues that we actually want on the homepage.